tests/collectors/hra/test_collector.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import mock
+import pytest
+import datetime
+import requests
+from collections import defaultdict
+from collectors.hra.collector import collect, _make_request_url
+
+
+class TestHRACollector(object):
+    def test_collect(self, conf, conn):
```

Comment:
I don't understand what this is testing. If we're checking that the data is being collected, this needs Betamax, and also needs to check the preconditions. What if the `conn['warehouse']['hra'].count()` was already bigger than 10?

Another suggestion is to use `@pytest.mark.skipif()` to write tests that should only be run on specific conditions (see https://docs.pytest.org/en/latest/skipping.html#id1)